src/BlazorWebView/src/Microsoft.AspNetCore.Components.WebView.Maui/Windows/BlazorMauiWebViewHandler.Windows.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.IO;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Maui.Handlers;
+using WebView2Control = Microsoft.UI.Xaml.Controls.WebView2;
+
+namespace Microsoft.AspNetCore.Components.WebView.Maui
+{
+	public partial class BlazorWebViewHandler : AbstractViewHandler<IBlazorWebView, WebView2Control>
+	{
+		private WebView2WebViewManager? _webviewManager;
+
+		protected override WebView2Control CreateNativeView()
+		{
+
+			//_webViewClient = GetWebViewClient();
+			//aWebView.SetWebViewClient(_webViewClient);
+
+			//_webChromeClient = GetWebChromeClient();
+			//aWebView.SetWebChromeClient(_webChromeClient);
+			return new WebView2Control();
+		}
+
+		protected override void DisconnectHandler(WebView2Control nativeView)
+		{
+			//nativeView.StopLoading();
+
+			//_webViewClient?.Dispose();
+			//_webChromeClient?.Dispose();
+		}
+
+		private bool RequiredStartupPropertiesSet =>
+			//_webview != null &&
+			HostPage != null &&
+			Services != null;
+
+		public string? HostPage { get; private set; }
+		public ObservableCollection<RootComponent>? RootComponents { get; private set; }
+		public new IServiceProvider? Services { get; private set; }
```

Comment:
Is there a way to make this services cooperative with our Services? Created a ServiceProvider scoped to this handler that can fallback to ours? Is that useful at all?

Reply:
Yes I want to share this! I didn't look yet at what it would take to make it all shared. It's probably trivial, I just didn't try it yet.